src/countriesList/countriesList.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Controller, Get } from '@nestjs/common';
+import { CountriesListService } from './countriesList.service';
+
+@Controller('countriesList')
```

Comment:
https://betterprogramming.pub/10-best-practices-for-naming-rest-api-endpoints-442ae592a3a0
https://restfulapi.net/resource-naming/